packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -74,6 +75,41 @@ const tabs: VerticalTabItemProps[] = [
       // { name: "embeds", href: "/v2/settings/developer/embeds" },
     ],
   },
+  {
+    name: "organization",
+    href: "/settings/organizations",
```

Comment:
shouldn't this url be singular? `/settings/organization`

Reply:
Likely so - we can handle this in a follow up as there are a few branches using this now